### DISCUSSION TITLE
KeyCloak-js instance not found

### DISCUSSION THREAD

**Discussion by Tagary** [keycloak/keycloak#35026] [Q&A]
I'm trying to do a logout from a page. The page with the keycloak is on another page.

The structure looks like this:

Page Keycloak, page App.

When I go to the keycloak I am redirected to the page App and I can logout from it because the iframe Keycloak is on the page. But after I reload the page keycloak null. And because of this I can't logout from the page, is there any request maybe in keycloak that would make a request and logout.

First loaded page:
![image](https://github.com/user-attachments/assets/5da064ef-786a-44dd-bb85-277aa1a4f524)
![image](https://github.com/user-attachments/assets/7714ec4e-bf9e-4bec-aa81-e1648e47f91d)

When reload page in redirected page:
![image](https://github.com/user-attachments/assets/637cb1de-407e-47d0-869f-6dbb86be0050)
![image](https://github.com/user-attachments/assets/6531ee0f-8c28-45d1-b753-beb530b0e1ca)

But i have token in LocalStorage. 
And not leave my page.

**Comment by jonkoops** [comment]
Its not going to be possible to answer this question without more details of your code. Consider creating a [Minimal, Reproducible Example](https://stackoverflow.com/help/minimal-reproducible-example) so people can help you.
